worker/games.py

```diff
@@ -81,6 +81,16 @@ def format_return_code(r):
     else:
         return str(r)
 
+def send_sigint(p):
+    try:
+        if p.poll() is None: # slight race condition
```

Comment:
what's a 'slight race condition' ? Might need some clearer comment.

Reply:
IMO the condition checks if the process is still running (exit status `None`), then send the signal, but the process could be exited just after the check.
The EAFP way could have the (low probability) risk to send the signal to a new process with the same pid.

Reply:
OK, now I understand.

Reply:
can't we use https://docs.python.org/3/library/subprocess.html#subprocess.Popen.send_signal

Reply:
Ah... I looked for send_signal, but in the signal module it is only in 3.10. I overlooked the subprocess module. I will change it.

Reply:
Ok I updated it. The code is now much simpler and robust obviously.